Running on desktop21:
Tue Jan  3 19:08:58 2023       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.33.01    Driver Version: 440.33.01    CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 108...  Off  | 00000000:03:00.0 Off |                  N/A |
| 13%   28C    P0    54W / 250W |      0MiB / 11178MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
              total        used        free      shared  buff/cache   available
Mem:         257560        1817      255477         134         265      254899
Swap:          4095           0        4095

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "6.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "6.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "7.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "7.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "8.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "8.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "9.py", line 3, in <module>
    import torchvision.transforms as transforms
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "9.py", line 3, in <module>
    import torchvision.transforms as transforms
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "9.py", line 3, in <module>
    import torchvision.transforms as transforms
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "9.py", line 3, in <module>
    import torchvision.transforms as transforms
ModuleNotFoundError: No module named 'torchvision'
Traceback (most recent call last):
  File "5.py", line 1, in <module>
    import ads3 as ads3
  File "/home/roba/ADS-Data-loading/ads3.py", line 9, in <module>
    import torchvision
ModuleNotFoundError: No module named 'torchvision'
